packages/react/src/runtime/sheet.tsx

```diff
@@ -141,5 +154,18 @@ export default function createStyleSheet(opts: StyleSheetOpts) {
     } else {
       style.appendChild(document.createTextNode(css));
     }
+
+    if (!ssrStyleElementsCleaned) {
+      // SSR step 2: Clean up the SSRd styles - we don't need them anymore
+      // as their client side counterparts have been applied above.
+      const ssrStyles = document.querySelectorAll('style[data-cmpld]');
+
+      for (let i = 0; i < ssrStyles.length; i++) {
+        const styleElement = ssrStyles[i];
+        styleElement.remove();
+      }
+
+      ssrStyleElementsCleaned = true;
+    }
```

Comment:
Bundlesize is going up. See if we can reduce it a bit. These steps are repeated:

```ts
const ssrStyles = document.querySelectorAll('style[data-cmpld]');

for (let i = 0; i < ssrStyles.length; i++) {
  const styleElement = ssrStyles[i];
}
```

Reply:
I think minifiers are smart enough to figure out such repeating, but still lets check once.